TicTacToe/test/tictactoe/TicTacToeTest.java

```diff
@@ -2,35 +2,139 @@
 
 import org.junit.Test;
 import static org.junit.Assert.*;
-
+/**
+ * 
+ * @author bachle
+ */
 public class TicTacToeTest {
 
+    /**
+     * Test of isWinner method, of class TicTacToe.
+     */
     @Test
     public void testIsWinner() {
+        //Horizontal
+        TicTacToe test1 = new TicTacToe();
+        for (int i = 0; i < 3; i++) {
+            test1.playMove('X', 0, i);
+        }
+        assertEquals(test1.isWinner('X'), true);
+
+        //Vertical
+        TicTacToe test2 = new TicTacToe();
+        for (int i = 0; i < 3; i++) {
+            test2.playMove('X', i, 0);
+        }
+        assertEquals(test2.isWinner('X'), true);
+
+        //Diagonal
+        TicTacToe test3 = new TicTacToe();
+        for (int i = 0; i < 3; i++) {
+            test3.playMove('O', i, i);
+        }
+        assertEquals(test3.isWinner('O'), true);
+
+        //Empty
+        TicTacToe test4 = new TicTacToe();
+        assertEquals(test4.isWinner('X'), false);
```

Comment:
Your only false checking is for an empty board.  What about boards that are partially full?  When X wins, you should also verify that O does NOT win.